src/posix/main.c

```diff
@@ -330,6 +344,7 @@ int main(int argc, char *argv[])
 #else
     otCliUartInit(instance);
 #endif
+    otCliSetUserCommands(&radioUrlCommand, 1, &config.mPlatformConfig);
```

Comment:
This is just for coverage purpose for now. I tend to keep `PosixConfig config` a temporary variable inside the `InitInstance()` function and revert the change to `InitInstance()`.

```suggestion
    otCliSetUserCommands(&radioUrlCommand, 1, argv[optind]);
```